glide.lock

```diff
@@ -1,87 +1,12 @@
 hash: b3ffe082966570570810f1f52c796da3dc31d24c84248da9aed9e06db26996ff
-updated: 2017-11-20T13:53:50.869715462-05:00
+updated: 2017-11-23T10:27:17.705741188+01:00
```

Comment:
This patch series shouldn't require an update to glide.lock. No build dependencies have changed.